client/state/data-layer/wpcom/read/lists/tags/new/index.js

```diff
@@ -10,7 +10,7 @@ import { http } from 'calypso/state/data-layer/wpcom-http/actions';
 import { dispatchRequest } from 'calypso/state/data-layer/wpcom-http/utils';
 import { errorNotice, successNotice } from 'calypso/state/notices/actions';
 import { READER_LIST_ITEM_ADD_TAG } from 'calypso/state/reader/action-types';
-import { receiveReaderListAddTag } from 'calypso/state/reader/lists/actions';
+import { receiveAddReaderListTag } from 'calypso/state/reader/lists/actions';
```

Comment:
This typo was introduced in #43173

@bluefuton could you review and confirm?

Reply:
Good catch @scinos! I've tested List Management and all looks good to me. ✅